examples/gcoap/Makefile

```diff
@@ -27,6 +27,22 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# DTLS configuration
+GCOAP_ENABLE_DTLS ?= 0
+ifeq (1,$(GCOAP_ENABLE_DTLS))
+	CFLAGS += -DCONFIG_GCOAP_ENABLE_DTLS=1
```

Comment:
Please add the same to [gcoap/Kconfig](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/application_layer/gcoap/Kconfig), as a `bool`. Macros in `CONFIG_` namespace are expected to be exposed via Kconfig.

Reply:
Was not my intention to not support Kconfig, just did not address it at this point. Will address Kconfig with the next fixups :)